### ISSUE TITLE
Existing "Album Artist" will not be cleared by "Import From File Tags" when that field is cleared by another tag editor

### ISSUE THREAD

**Issue by atskler** [mixxxdj/mixxx#11241]
### Bug Description

Existing "Album Artist" will not be cleared by "Track > Right click > Metadata > Import From File Tags" when that field is cleared by another tag editor. (This works correctly with the "Comment" field.)

### Version

2.3.3-121-gd1f63374fc

### OS

OpenSUSE Tumbleweed

**Comment by daschuer** [comment]
I think we need to treat a missing "Album Artist" as an empty string and not as "unknown" null value where the Mixxx value is adopted.

**Comment by uklotzde** [comment]
> I think we need to treat a missing "Album Artist" as an empty string and not as "unknown" null value where the Mixxx value is adopted.

No. The check for reading ID3v2 comment frames is wrong, should be `isEmpty()` instead of `isNull()` for consistency. Resetting missing/empty properties on import is controlled by `resetMissingTagMetadata`.

**Comment by uklotzde** [comment]
The only feature that is missing is a UI option for `kResetMissingTagMetadataOnImportConfigKey`.

This is also a legit reason for keeping the *Easy* tag on this issue.
